sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.            "EndpointConfigurations": [{"InstanceType": "ml.m5.2xlarge"}],
</pre><p>Why do you only want to use one instance type here, versus the list of 9 instance types for the non-advanced job?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Added details about why I am doing that. The purpose is to find out maximum number of invocations per minimum latency of 50 ms.